.github/renovate.json

```
@@ -12,5 +12,10 @@
       "matchPackageNames": ["com.google.protobuf"],
       "allowedVersions": "<=0.8.19"
     }
-  ]
+  ],
+  "postUpgradeTasks": {
+    "commands": [
+      "./gradlew app:dependencyGuardBaseline app-nia-catalog:dependencyGuardBaseline"
```

nit: you don't need to specify the modules. The tasks will be available on all module that have applied the plugin.

```suggestion
      "./gradlew dependencyGuardBaseline"
```

---

Thanks, that's much cleaner.